sirepo/package_data/static/js/radia.js

```diff
@@ -3758,36 +3742,113 @@ SIREPO.app.directive('shapeSelector', function(appState, panelState, plotting, r
 SIREPO.viewLogic('objectShapeView', function(appState, panelState, radiaService, $scope) {
     let ctr = [];
     let modelType = null;
+    let parent = $scope.$parent;
     let size = [];
-    let geomIndices = {};
 
     $scope.modelData = appState.models[$scope.modelName];
+    size = $scope.modelData.size.split(/\s*,\s*/).map((x) => {
```

Comment:
I think the code would be clearer if you use a named function for the split, map, parseFloat and passed in the string.

Reply:
To be clear I mean something like

```js
function myNewFunctionWithAGreatName(str) {
  return str.split(/\s*,\s*/).map(x => parseFloat);
}
```

Reply:
I'll make a separate issue for it